providers/frame/polkadot_dev.ts

```diff
@@ -1,13 +1,27 @@
+import { File } from "../../codegen/frame/mod.ts"
 import * as $ from "../../deps/scale.ts"
 import { Env, PathInfo } from "../../server/mod.ts"
+import { fromPathInfo } from "../../server/PathInfo.ts"
 import { getAvailable } from "../../util/port.ts"
 import { FrameBinProvider } from "./FrameBinProvider.ts"
+import { createCustomChainSpec } from "./utils/mod.ts"
+
+const DEV_RUNTIME_PREFIXES = {
+  polkadot: 0,
+  kusama: 2,
+  westend: 42,
+  rococo: 42,
+} as const
+
+const TEST_USER_COUNT = 10
 
 export interface PolkadotDevProviderProps {
   polkadotPath?: string
 }
 
 export class PolkadotDevProvider extends FrameBinProvider {
+  #testUserCountCache: Record<string, number> = {}
```

Comment:
We use `Cache` in other places; let's rename this to avoid conflation
```suggestion
  #testUserCount: Record<string, number> = {}
```

Reply:
No reason to even prefix with `test` imo. Should be clear without.